pkg/arvo/sys/hoon.hoon

```diff
@@ -10178,10 +10182,13 @@
       ~
     =/  dov/?(~ ^)
       =/  dab/(map term hoon)  q.q.n.dom
+      =/  dag  (get-arms dog p.n.dom)
+      =.  dag  (arms-check dab dag)
```

Comment:
Another option here (and above) would be to combine some of these calls:

```
=/  gol  
=/  log  (chapters-check (core-check gol))
...
=/  dag  (arms-check dab (get-arms dog p.n.dom))
```

I realize this nit-picky -- no strong feelings on my part ...